### DISCUSSION TITLE
Update table

### DISCUSSION THREAD

**Discussion by madeloop** [uwdata/arquero#313] [Q&A]
Is Arquero able to handle table updates? My use case would require lots of streaming updates.

I found one other post asking the same question last year and it never got answered. I don't see anything in the docs related to updating tables.

**Comment by StephM-W** [comment]
I'd love to be able to stream updates to Arquero too. If we're asking for the moon, having reactive joins would be nirvana (updates to either source tables would also update the join result)

So far all I've found that talks to the topic is this response from @jheer, https://github.com/uwdata/arquero/issues/38#issuecomment-738409902 and its associated observable notebook: https://observablehq.com/@uwdata/arquero-row-manipulation
